osquery/tables/events/linux/seccomp_events.cpp

```diff
@@ -0,0 +1,522 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#include <osquery/core/flags.h>
+#include <osquery/events/linux/auditeventpublisher.h>
+#include <osquery/logger/logger.h>
+#include <osquery/registry/registry_factory.h>
+#include <osquery/tables/events/linux/seccomp_events.h>
+#include <osquery/utils/system/uptime.h>
+
+namespace osquery {
+
+const std::unordered_map<std::uint64_t, std::string>
```

Comment:
Can these be moved at the end of the file? It would make it more readable by showing the class methods first